fe/fe-core/src/main/java/com/starrocks/connector/iceberg/IcebergPartitionUtils.java

```diff
@@ -78,11 +78,11 @@ public static Set<IcebergPartition> getChangedPartition(Table table, long fromTi
         if (snapShot.timestampMillis() >= fromTimestampMillis) {
```

Comment:
if iceberg roll back snapshot, it will be bigger than snapShot.timestampMillis().  fromTimestampMillis 
 use System.currentTimeMillis() to compute, not affected by time zone